app/config/campaigns.yml

```diff
@@ -59,4 +59,17 @@ campaigns:
       default_bucket: "new_layout"
       url_key: confirmation
       active: true
-      param_only: false
\ No newline at end of file
+      param_only: false
+
+    encryption_hint:
+      description: Testing users seeing/not seeing the hint about encrypted connection when they submit a form.
+      reference: "https://phabricator.wikimedia.org/T286446"
+      start: "2021-07-29"
+      end: "2021-08-12"
+      buckets:
+        - "visible"
+        - "not_visible"
+      default_bucket: "visible"
+      url_key: encrypthint
```

Comment:
When we introduced the bucket testing feature, we used 1-3 character URL keys. This has the benefit of

* fewer bytes to transfer in the banners
* easier to type in URLs when testing manually
* obfuscating the A/B test

I don't have a strong opinion, just wanted to point that out